src/gui/entry/EntryView.h

```diff
@@ -38,31 +43,83 @@ class EntryView : public QTreeView
     void setCurrentEntry(Entry* entry);
     Entry* entryFromIndex(const QModelIndex& index);
     void setEntryList(const QList<Entry*>& entries);
-    bool inEntryListMode();
+    bool inSearchMode();
     int numberOfSelectedEntries();
     void setFirstEntryActive();
 
+    /**
+     * @author Fonic <https://github.com/fonic>
+     * Methods to get/set state of 'Hide Usernames'/'Hide Passwords' settings
+     * (NOTE: these are just pass-through methods to avoid exposing entry model)
+     */
+    bool hideUsernames() const;
+    void setHideUsernames(const bool hide);
+    bool hidePasswords() const;
+    void setHidePasswords(const bool hide);
+
+    /**
+     * @author Fonic <https://github.com/fonic>
+     * Methods to get/set state of view
```

Comment:
Move to CPP file. Generally, I also think you shouldn't state that this is a method or signal or what not. Just use a simple present-tense sentence about what it does (same as in commit messages).